january-slates/readme.md

```diff
@@ -1 +1,106 @@
+cpb-aacip-41-34fn32g7,",","Carolina Journal          ,","00:00:00;00 ,","00:00:14;28 ,","typed        ,",digital?,,,",",key-value pairs,",",countdown,
+```
+FINAL PREPARED GOLD OUTPUT - `.tsv` file that seems to be actually comma-separated anyway. The gold files conform to the repository readme guideline that each 
+gold must relate to only one GUID. Therefore, each of these gold files is only 1 video/GUID each. Note: the column headers are added to each file.  
+```
+FORMAT & EXAMPLE:
+GUID,Series/Group,Slate Start,Slate End,Writing Types,Recorded/Digital,format of most of the information,Anything moving on screen during slate?
+cpb-aacip-29-01pg4g2x,Prime Time Wisconsin,00:00:00;00,00:00:08;14,typed,digital,key-value pairs,no
+```
+
+## Tool Installation: Unknown
+_TODO: Tool used is unknown. Follow up questions with Sammy should illuminate this._
+
+## Annotation Guidelines: Transcribing/Closed Captioning
+_What guidelines were used for this?_
+### Preparation
+Multiple videos must be prepared for annotation.  
+### Process of Annotating
+
+Per column:  
+* GUID - the AAPB id for that video e.g. "cpb-aacip-81-881jx33t". 
+CPB is the [Corporation for Public Broadcasting](https://cpb.org/faq#1-1:~:text=Public%20Broadcasting%20(CPB)%3F-,CPB,-is%20a%20private).
+"aacip" is likely a collection name (unverified).
+The first number seems to be series number. Eg. 81 is "Woman", and "29" is both Prime Time Wisconsin" and "Wisconsin Week".
+The final number which includes letters is the unique guid number.  
+* (comma) - column of commas: ","  
+* Series/Group              ,- what tv series or group this video belongs to.  
+* Slate Start ,- When the slate starts appearing. (See Decisions). Format likely "hr:mn:se:fr" out of 30 fps.     
+* Slate End   ,- When the slate stops being shown on screen. (See Decisions)  
+* Writing Types,- Slates contain written information. Much of this material is from the early days of tv. eg. "handwritten", "typed" or "other"   
+* Recorded/Digital - _unknown what this column pertains to_  
+* (empty)  
+* (empty)  
+* (comma)  
+* format of most of the information - visually how is the textual information presented in the slate? e.g. "boxes to fill in", "key-value pairs", "free text"  
+* (comma)  
+* Anything moving on screen during slate? - are there things like animations and other things that are moving during the slate information
+
+_Note: in some columns there is a comma within the raw value of the column._
+
+### Decisions and Differentiations during Annotation
+**What denotes start and end** - Decisions as to what exactly constitutes the start and end of the slate times is unknown. Is it when it first starts fading in, or when its fully done fading? 
+
+**Errors in the raw** - There are errors in the raw format, notably, in `CLAMS_slate_annotation_metadata.csv` line 203 "cpb-aacip-394-150gbd75", column(Writing Types F) is a typo "typeed" instead of "typed". 
```

Comment:
Probably a good idea to create a hyperlink to that line of that file of that commit as a permalink. e.g. `https://github.com/clamsproject/aapb-annotations/blob/f884e10d0b9d4b1d68e294d83c6e838528d2c249/january-slates/230101-aapb-collaboration-7/CLAMS_slate_annotation_metadata.csv?plain=1#L203`